src/record.py

```diff
@@ -3,16 +3,86 @@
 
 
 class Field:
-    pass
+    def __init__(self, value):
+        self._value = value
+
+    def __str__(self):
+        return str(self._value)
+
+    @property
+    def value(self):
+        return self._value
 
+    @value.setter
+    def value(self, new_value):
+        self._value = new_value
 
 class Name(Field):
     pass
```

Comment:
Please add logic to this class

Reply:
Name class in my program can remain unchanged. I've already inherited the Field class for it and don't need any additional methods or properties for the Name class. The Name class is used to represent the name of a contact and stores its value. My code already includes all the provisioning methods from the Field class, such as __str__, @property, @value.setter, as the Field class defines the general logic to manage the value of the field.